letsencrypt/cli.py

```diff
@@ -267,16 +267,30 @@ def _treat_as_renewal(config, domains):
     return None
 
 
-def _report_new_cert(cert_path):
-    """Reports the creation of a new certificate to the user."""
+def _report_new_cert(cert_path, fullchain_path):
+    """
+    Reports the creation of a new certificate to the user.
+    :param string cert_path: path to cert
```

Comment:
And nit: I think `:param string` should be `:param str` to keep with the rest of the codebase.